### DISCUSSION TITLE
Global search only in visible columns

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#5765] [Help]
Hi

Is there any way to define that the global search should only work against columns that are visible? Right now I can hide columns, but they are still searchable.

**Comment by danharrin** [comment]
You mean with the toggleable columns, right?

**Reply by oliverbj** [reply]
Yes, exactly.. So I have these two columns hidden ("Shipper", "Consignee"), but I can still search for values in those columns.

![image](https://user-images.githubusercontent.com/7851696/220904142-bdd36d7a-44cd-4039-8ebf-edc69914ab74.png)

**Reply by danharrin** [reply]
Hmm I don't think this is expected behaviour for most people though

**Reply by danharrin** [reply]
Feel free to make a PR to enable this option, but I won't work on this feature since its not something I'd use.
